_includes/homeslider.html

```diff
@@ -0,0 +1,35 @@
+</div>
+
+<div id="home-slider-wrapper"><div class="container">
+      <div>
+            <h2 class="center-text">IRCv3 Features</h2>
+
+            <ul class="homepage-features">
+                  <li>Standardised account login using SASL to speed up registration and authentication. <a href="http://ircv3.net/specs/extensions/sasl-3.1.html"><sup>[3.1]</sup></a> <a href="http://ircv3.net/specs/extensions/sasl-3.2.html"><sup>[3.2]</sup></a></li>
+                  <li>Providing the account information of other clients for the development of more advanced client features. <a href="http://ircv3.net/specs/extensions/account-notify-3.1.html"><sup>[1]</sup></a> <a href="http://ircv3.net/specs/extensions/account-tag-3.2.html"><sup>[2]</sup></a> <a href="http://ircv3.net/specs/extensions/extended-join-3.1.html"><sup>[3]</sup></a></li>
+                  <li>Optional metadata able to be attached to each message for easier, standardised extension development. <a href="http://ircv3.net/specs/core/message-tags-3.2.html"><sup>[link]</sup></a></li>
+                  <li>Grouping related messages to simplify the display of those messages to users. <a href="http://ircv3.net/specs/extensions/batch-3.2.html"><sup>[link]</sup></a></li>
+            </ul>
+      </div>
+      <div>
+            <h2 class="center-text">What We're Working On</h2>
+
+            <ul class="homepage-features">
+                  <li>Standardised account registration and verification, allowing clients to provide better interfaces for end users. <a href="https://github.com/ircv3/ircv3-specifications/pull/152"><sup>[link]</sup></a></li>
+                  <li>Giving clients a standardised way to recognise, access and view chat history (provided by bouncers or servers). <a href="http://ircv3.net/specs/extensions/batch/chathistory-3.3.html"><sup>[link]</sup></a></li>
+                  <li>Providing a more standardised alternative to the <span class="mono">STATS</span> command that is consistent across server software. <a href="https://github.com/ircv3/ircv3-specifications/issues/99"><sup>[link]</sup></a></li>
```

Comment:
I don't think this should be mentioned here, we aren't working on this actively and it isn't interesting to regular users as STATS is restricted to opers. Could mention providing mechanisms to secure users (and providing best practices, when we do that) or roaming instead which I believe is interesting to more people.